### PR TITLE
PHP server generator ze-ph (Zend Expressive + Path Handler)

### PR DESCRIPTION
**Pull request by Articus**
### PR checklist

- [+] Read the [contribution guildelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [+] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates)
- [+] Filed the PR against the correct branch: master for non-breaking changes and `2.3.0` branch for breaking (non-backward compatible) changes.

### Description of the PR

New PHP server stub generator using [Zend Expressive](https://zendframework.github.io/zend-expressive) and [Path Handler](https://github.com/Articus/PathHandler).

### CODE REVIEW

**Comment by wing328** [comment]
@Articus thanks for the contribution. The PR looks good to me.

2 minor suggestions:
 - create a new branch instead of using master as per git best practices
 - update README.md.mustache with instruction on how to run the generated-code

File: bin/windows/ze-ph-petstore.bat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+set executable=.\modules\swagger-codegen-cli\target\swagger-codegen-cli.jar
+
+If Not Exist %executable% (
+  mvn clean package
+)
+
+REM set JAVA_OPTS=%JAVA_OPTS% -Xmx1024M
+set ags=generate -i modules\swagger-codegen\src\test\resources\2_0\petstore.json -l ze-ph -o samples\client\petstore\ze-ph
```
Comment (wing328) [inline bin/windows/ze-ph-petstore.bat]: FYI. I'll update this line to use modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml and output the generated code under `samples\server\petstore\ze-ph` instead

**Comment by wing328** [comment]
cc @arnested

File: bin/windows/ze-ph-petstore.bat (hunk 1 above)
Comment (Articus) [inline bin/windows/ze-ph-petstore.bat]: Sure, thanks for noticing) And sorry for not checking this "copy-paste" from bin/windows/php-petstore.bat thoroughly. For actual testing I used IDEA's "Run/Debug configuration".

**Comment by wing328** [comment]
Tweet: https://twitter.com/wing328/status/820995831058735105

Please help retweet.

I also updated README (swagger codegen) to list you as the template creator.

**Pull request merged**